internal/pkg/agent/cmd/otel_flags.go

```diff
@@ -28,6 +29,11 @@ func setupOtelFlags(flags *pflag.FlagSet) {
 	flags.StringArray(otelSetFlagName, []string{}, "Set arbitrary component config property. The component has to be defined in the config file and the flag"+
 		" has a higher precedence. Array config properties are overridden and maps are joined. Example --set=processors.batch.timeout=2s")
 
+	flags.Bool(otelSetSupervised, false, "Set the that this collector is running as supervised.")
```

Comment:
```suggestion
	flags.Bool(otelSetSupervised, false, "Set that this collector is running as supervised.")
```

Reply:
reworded in https://github.com/elastic/elastic-agent/pull/8248/commits/9887d603c22242d295618b90a5cca3e8b8199168